2025-03-18 16:09:31,635 [INFO] USER:thattepallyjayasreet s3_resource.py:40:validate_bucket_name LOG: Starting validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-6675'
2025-03-18 16:09:31,636 [INFO] USER:thattepallyjayasreet s3_resource.py:73:validate_bucket_name LOG: Finished validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-6675'
2025-03-18 16:09:31,637 [INFO] USER:thattepallyjayasreet helper.py:525:convert LOG: Stripping run_id:JAP-11,run_type:stm,type:user,user_id:5638a064 from "," a bit..
2025-03-18 16:09:31,637 [INFO] USER:thattepallyjayasreet helper.py:529:convert LOG: Converting: run_id:JAP-11,run_type:stm,type:user,user_id:5638a064 to dict..
2025-03-18 16:09:31,653 [INFO] USER:thattepallyjayasreet helper.py:541:convert LOG: Converted to such a dict: {'run_id': 'JAP-11', 'run_type': 'stm', 'type': 'user', 'user_id': '5638a064'}
2025-03-18 16:09:33,628 [WARNING] USER:thattepallyjayasreet generator.py:76:generate_configuration_files LOG: The config_path property is not specified. The working directory is used
2025-03-18 16:09:33,628 [ERROR] USER:thattepallyjayasreet __init__.py:44:_mkdir LOG: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\thattepallyjayasreet\\aws\\task03\\.syndicate-config-dev'
2025-03-18 16:09:33,628 [WARNING] USER:thattepallyjayasreet generator.py:93:generate_configuration_files LOG: The "project_path" property is not specified. The working directory will be used as a project path. To change the path, edit the syndicate.yml by path C:\Users\thattepallyjayasreet\aws\task03\.syndicate-config-dev
2025-03-18 16:09:33,649 [INFO] USER:thattepallyjayasreet generator.py:155:generate_configuration_files LOG: Syndicate initialization has been completed. 
Set SDCT_CONF:
Unix: export SDCT_CONF="C:\Users\thattepallyjayasreet\aws\task03\.syndicate-config-dev"
Windows: setx SDCT_CONF C:\Users\thattepallyjayasreet\aws\task03\.syndicate-config-dev
2025-03-18 16:09:33,652 [INFO] USER:thattepallyjayasreet helper.py:330:timed LOG: Stage config, elapsed time: 0:00:01.999757
